2.16.0 syntax error

Hey!

I have a workflow set up with some basic pyhton test file that runs with pytest and generates a junit.xml file. I have been using EnricoMi for about a year now to publish results with no issues, but since the update for 2.16.0 hit, I've been running into the following error:
![image](https://github.com/EnricoMi/publish-unit-test-result-action/assets/68228923/35aa3d3f-a49e-4365-abfd-a948322b15c9)

```
/c/Users/ADF VM/AppData/Local/Programs/Python/Python310/python3
Python that creates venv: C:\Users\ADF VM\AppData\Local\Programs\Python\Python310\python3.exe
C:\Users\ADF: line 1: syntax error near unexpected token `('
C:\Users\ADF: line 1: `[26B8:26C8][2023-06-15T14:09:00]i001: Burn v3.10.4.4718, Windows v10.0 (Build 22621: Service Pack 0), path: C:\Windows\Temp\{92DCA7E6-3972-44A9-AE81-C061C4779222}\.cr\VC_redist.x[64](https://github.com/adfdesktop/launch-autotests/actions/runs/8376472318/job/22936250951#step:2:66).exe'
  Error: Process completed with exit code 2.
```

The issue can be solved by manually setting the version of the action to the previous one, however it'd probably be best to keep using the latest one.
```yaml
uses: EnricoMi/publish-unit-test-result-action/windows/bash@v2.15.1
```

I'm running Python 3.10.9 on a self-hosted runner with Windows 11, not sure what other info to provide. My yaml is below. Thanks!

```yaml
name: TESTS
on:
  workflow_dispatch:
permissions: write-all
jobs:
  drive-tests:
    runs-on: [self-hosted, drive-image-tests]
    timeout-minutes: 1440
    steps:
      - name: Run tests
        shell: powershell
        run: |
          cd 'D:/tests'
          pytest .\test.py -s --tb=no --junitxml=junit.xml
          
      - name: Publish test results
        uses: EnricoMi/publish-unit-test-result-action/windows/bash@v2
        if: always()
        with:
          files: |
            D:/tests/junit.xml
```

--- a/windows/bash/action.yml
+++ b/windows/bash/action.yml
@@ -176,7 +176,7 @@ runs:
         echo "Python that creates venv: $PYTHON_BIN"
         echo "PYTHON_BIN=$PYTHON_BIN" >> "$GITHUB_ENV"
 
-        PYTHON_VERSION="$($PYTHON_BIN -c 'import sys; print(f"{sys.version_info.major}.{sys.version_info.minor}")')"
+        PYTHON_VERSION="$("$PYTHON_BIN" -c 'import sys; print(f"{sys.version_info.major}.{sys.version_info.minor}")')"
         if [[ "$PYTHON_VERSION" == "3.7" ]]
         then
           echo "DEPENDENCIES_VERSION=3.7" >> "$GITHUB_ENV"

